akka-stream-tests-tck/src/test/scala/akka/stream/tck/Timeouts.scala

```diff
@@ -12,8 +12,8 @@ import akka.testkit._
  */
 object Timeouts {
 
-  def publisherShutdownTimeoutMillis: Int = 1000
+  def publisherShutdownTimeoutMillis: Int = 3000
 
-  def defaultTimeoutMillis: Int = 500
+  def defaultTimeoutMillis: Int = 800
```

Comment:
This adds 100s to validation on my machine so I am not happy. The current limit is too tight, increasing it will make validation almost 2 minutes slower... Is there any viable solution?